features/networking/sdn.feature

```diff
@@ -33,18 +33,36 @@ Feature: SDN related networking scenarios
     And the output should contain "Using userspace Proxier"
     """
 
-  # @author bmeng@redhat.com
+  # @author rbrattai@redhat.com
   # @case_id OCP-11286
   @admin
   @destructive
   Scenario: iptables rules will be repaired automatically once it gets destroyed
+    # we do not detect incomplete rule removal since ~4.3, BZ-1810316
+    # so only test on >= 4.3
+    Given the master version >= "4.3"
     Given I select a random node's host
-    And the node iptables config is verified
+    And the node iptables config is checked
+    And the step succeeded
     And the node service is restarted on the host after scenario
+    And I register clean-up steps:
+    """
+    When the node iptables config is checked
+    Then the step succeeded
+    """
+
+    When the node standard iptables rules are removed
+    And 35 seconds have passed
+    When the node iptables config is checked
+    # Removing individual rules will not trigger automatic repair on < 4.3
+    # the check should fail
+    Then the step failed
+    # >= 4.3 we have to flush all the rules and tables to trigger a repair
+    When the node standard iptables rules are completely flushed
+    And 35 seconds have passed
```

Comment:
I'll try replacing this with a `And I wait up to 35 seconds for the steps to pass:`